src/osmet/mod.rs

```diff
@@ -150,6 +151,23 @@ pub fn osmet_unpack(config: &OsmetUnpackConfig) -> Result<()> {
     Ok(())
 }
 
+pub fn find_matching_osmet_in_dir(sector_size: NonZeroU32, osmet_dir: &Path) -> Result<PathBuf> {
+    for entry in WalkDir::new(osmet_dir) {
```

Comment:
Are we intentionally recursing into subdirs here?  We might not want to commit to that.  (The directory format isn't really a public API, so maybe this doesn't matter.)